recipes-kernel/kexec/kexec-tools_2.0.21.bbappend

```diff
@@ -1,9 +1,14 @@
 SUMMARY = "nilrt specific modification for kdump"
```

Comment:
This doesn't make a whole lot of sense. The `SUMMARY` assignment here is going to overwrite the package summary for the `kexec-tools` bb recipe. It would make more sense to create a new subpackage like:
```
FILESEXTRAPATHS:prepend := "${THISDIR}/${BPN}:"

SRC_URI += "file://kdump.conf"

PACKAGE_BEFORE_PN += "${PN}-nilrt"

RDEPENDS:${PN} += "${PN}-nilrt"

# ${PN}-nilrt
SUMMARY:${PN}-nilrt = "${SUMMARY} - NILRT bootargs"
pkg_postinst:${PN}-nilrt () {
    sed -i 's/set otherbootargs="\(.*\)"/set otherbootargs="\1 crashkernel=128M "/' /boot/runmode/bootimage.cfg
}
RDEPENDS:${PN}-nilrt += "${PN}"
```

Reply:
Comment retracted. This recipe is weird, and I need some time to think about how to architect it better.